assembl/static2/js/app/components/navbar/languageMenu.jsx

```diff
@@ -73,7 +75,7 @@ class LanguageMenu extends React.Component {
   };
 
   render() {
-    const { size, i18n, style, className, setWidth } = this.props;
+    const { size, i18n, style, className, setWidth = doNothing } = this.props;
```

Comment:
Did you have an error? Why do you need a default value now?

Reply:
yes. of the two uses of it, one doesn't provide this callback. I think it worked before but it was spewing red in the console.